### PR TITLE
Changing server connect to be consistent

### PR DESCRIPTION
**Pull request by potatoqualitee**

### CODE REVIEW

File: functions/Get-DbaBackupDevice.ps1 [hunk 1]
```diff
@@ -41,7 +44,8 @@ Returns all Backup Devices for the local and sql2016 SQL Server instances
 	Param (
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
 		[DbaInstanceParameter]$SqlInstance,
-		[System.Management.Automation.PSCredential]$SqlCredential
+		[System.Management.Automation.PSCredential]$SqlCredential,
```
Comment (wsmelton) [inline functions/Get-DbaBackupDevice.ps1]: This should be set to `[PSCredential][System.Management.Automation.CredentialAttribute()]`.

Comment (wsmelton) [inline functions/Get-DbaBackupDevice.ps1]: This is for all of them if you wanted to go ahead and update.

Comment (potatoqualitee) [inline functions/Get-DbaBackupDevice.ps1]: thanks!

**Comment by potatoqualitee** [comment]
yesssss
thanks!

**Pull request merged**